tasks/broad/ImputationTasks.wdl

```diff
@@ -307,25 +331,26 @@ task OptionalQCSites {
     File input_vcf
     File input_vcf_index
     String output_vcf_basename
-    String bcftools_vcftools_docker
+    String bcftools_vcftools_docker = "us.gcr.io/broad-dsde-methods/imputation_bcftools_vcftools_docker:v1.0.0"
     Float? optional_qc_max_missing
     Float? optional_qc_hwe
+    Int cpu = 1
```

Comment:
in other places it looks like the docker variables (docker, cpu, memory and disk) are below the other task inputs and separated by a blank line. is that a style we want to implement while cleaning this up? I personally like the look, but probably just a preference not really important

Reply:
i like the look of that as well, it helps to separate runtime attribute inputs from the others. will update 👍